sway/commands/unmark.c

```diff
@@ -0,0 +1,32 @@
+#include <string.h>
+#include <strings.h>
+#include "sway/commands.h"
+#include "list.h"
+#include "stringop.h"
+
+struct cmd_results *cmd_unmark(int argc, char **argv) {
+	swayc_t *view = get_focused_container(&root_container);
+
+	if (view->marks) {
+		if (argc) {
+			char *mark = join_args(argv, argc);
+			int index;
+			if (-1 != (index = list_seq_find(view->marks, (int (*)(const void *, const void *))strcmp, mark))) {
+				free(view->marks->items[index]);
+				list_del(view->marks, index);
+
+				if (0 == view->marks->length) {
```

Comment:
Constant value before the variable in the comparison.
 E.g: "0 equals mark's length" instead of "mark's length equals 0"